[2023-11-22T21:05:45.078+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: COVID-19.execute_sql_script manual__2023-11-22T19:29:07.263785+00:00 [queued]>
[2023-11-22T21:05:45.089+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: COVID-19.execute_sql_script manual__2023-11-22T19:29:07.263785+00:00 [queued]>
[2023-11-22T21:05:45.090+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-11-22T21:05:45.108+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): execute_sql_script> on 2023-11-22 19:29:07.263785+00:00
[2023-11-22T21:05:45.116+0000] {standard_task_runner.py:57} INFO - Started process 219 to run task
[2023-11-22T21:05:45.118+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'COVID-19', 'execute_sql_script', 'manual__2023-11-22T19:29:07.263785+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpndc3teby']
[2023-11-22T21:05:45.121+0000] {standard_task_runner.py:85} INFO - Job 51: Subtask execute_sql_script
[2023-11-22T21:05:45.175+0000] {task_command.py:416} INFO - Running <TaskInstance: COVID-19.execute_sql_script manual__2023-11-22T19:29:07.263785+00:00 [running]> on host 6c2240d4b080
[2023-11-22T21:05:45.446+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='COVID-19' AIRFLOW_CTX_TASK_ID='execute_sql_script' AIRFLOW_CTX_EXECUTION_DATE='2023-11-22T19:29:07.263785+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-22T19:29:07.263785+00:00'
[2023-11-22T21:05:45.449+0000] {sql.py:274} INFO - Executing: -- Table des dimensions
CREATE TABLE IF NOT EXISTS Departement (
    num_departement INT NOT NULL,
    nom_departement VARCHAR(255) NOT NULL,
    nom_region VARCHAR(255) NOT NULL,
    PRIMARY KEY (num_departement)
);

CREATE TABLE IF NOT EXISTS Passage (
    id_passage INT NOT NULL,
    nombre_passage_corona INT,
    nombre_passage_total INT,
    nombre_passage_corona_h INT,
    nombre_passage_corona_f INT,
    nombre_passage_total_h INT,
    nombre_passage_total_f INT,
    PRIMARY KEY (id_passage)
);

CREATE TABLE IF NOT EXISTS Hospitalisation (
    id_hospitalisation INT NOT NULL,
    nombre_hospitalisation_corona INT,
    nombre_hospitalisation_corona_h INT,
    nombre_hospitalisation_corona_f INT,
    PRIMARY KEY (id_hospitalisation)
);

CREATE TABLE IF NOT EXISTS Age (
    id_age INT NOT NULL,
    tranche_age VARCHAR(255) NOT NULL,
    PRIMARY KEY (id_age)
);

-- Table des faits
CREATE TABLE IF NOT EXISTS Urgence_covid (
    -- id_urgence_covid INT NOT NULL,
    -- id_hospitalisation INT FOREIGN KEY REFERENCES Hospitalisation(id_hospitalisation),
    -- id_passage INT FOREIGN KEY REFERENCES Passage(id_passage),
    -- id_age INT FOREIGN KEY REFERENCES Age(id_age),
    -- num_departement INT FOREIGN KEY REFERENCES Departement(num_departement),
    -- PRIMARY KEY (id_urgence_covid)
    id_urgence_covid INT NOT NULL,
    id_hospitalisation INT,
    id_passage INT,
    id_age INT,
    num_departement INT,
    PRIMARY KEY (id_urgence_covid),
    FOREIGN KEY (id_hospitalisation) REFERENCES Hospitalisation(id_hospitalisation),
    FOREIGN KEY (id_passage) REFERENCES Passage(id_passage),
    FOREIGN KEY (id_age) REFERENCES Age(id_age),
    FOREIGN KEY (num_departement) REFERENCES Departement(num_departement)
);

INSERT INTO Age VALUES (0, "tous âges");
INSERT INTO Age VALUES (1, "0-4 ans");
INSERT INTO Age VALUES (2, "5-14 ans");
INSERT INTO Age VALUES (3, "15-44 ans");
INSERT INTO Age VALUES (4, "45-64 ans");
INSERT INTO Age VALUES (5, "65-74 ans");
INSERT INTO Age VALUES (6, "75 ans et plus");
[2023-11-22T21:05:45.460+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-22T21:05:45.471+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-22T21:05:45.488+0000] {sql.py:418} INFO - Running statement: -- Table des dimensions
CREATE TABLE IF NOT EXISTS Departement (
    num_departement INT NOT NULL,
    nom_departement VARCHAR(255) NOT NULL,
    nom_region VARCHAR(255) NOT NULL,
    PRIMARY KEY (num_departement)
);

CREATE TABLE IF NOT EXISTS Passage (
    id_passage INT NOT NULL,
    nombre_passage_corona INT,
    nombre_passage_total INT,
    nombre_passage_corona_h INT,
    nombre_passage_corona_f INT,
    nombre_passage_total_h INT,
    nombre_passage_total_f INT,
    PRIMARY KEY (id_passage)
);

CREATE TABLE IF NOT EXISTS Hospitalisation (
    id_hospitalisation INT NOT NULL,
    nombre_hospitalisation_corona INT,
    nombre_hospitalisation_corona_h INT,
    nombre_hospitalisation_corona_f INT,
    PRIMARY KEY (id_hospitalisation)
);

CREATE TABLE IF NOT EXISTS Age (
    id_age INT NOT NULL,
    tranche_age VARCHAR(255) NOT NULL,
    PRIMARY KEY (id_age)
);

-- Table des faits
CREATE TABLE IF NOT EXISTS Urgence_covid (
    -- id_urgence_covid INT NOT NULL,
    -- id_hospitalisation INT FOREIGN KEY REFERENCES Hospitalisation(id_hospitalisation),
    -- id_passage INT FOREIGN KEY REFERENCES Passage(id_passage),
    -- id_age INT FOREIGN KEY REFERENCES Age(id_age),
    -- num_departement INT FOREIGN KEY REFERENCES Departement(num_departement),
    -- PRIMARY KEY (id_urgence_covid)
    id_urgence_covid INT NOT NULL,
    id_hospitalisation INT,
    id_passage INT,
    id_age INT,
    num_departement INT,
    PRIMARY KEY (id_urgence_covid),
    FOREIGN KEY (id_hospitalisation) REFERENCES Hospitalisation(id_hospitalisation),
    FOREIGN KEY (id_passage) REFERENCES Passage(id_passage),
    FOREIGN KEY (id_age) REFERENCES Age(id_age),
    FOREIGN KEY (num_departement) REFERENCES Departement(num_departement)
);

INSERT INTO Age VALUES (0, "tous âges");
INSERT INTO Age VALUES (1, "0-4 ans");
INSERT INTO Age VALUES (2, "5-14 ans");
INSERT INTO Age VALUES (3, "15-44 ans");
INSERT INTO Age VALUES (4, "45-64 ans");
INSERT INTO Age VALUES (5, "65-74 ans");
INSERT INTO Age VALUES (6, "75 ans et plus");, parameters: None
[2023-11-22T21:05:45.562+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 392, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedColumn: column "tous âges" does not exist
LINE 54: INSERT INTO Age VALUES (0, "tous âges");
                                    ^

[2023-11-22T21:05:45.578+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=COVID-19, task_id=execute_sql_script, execution_date=20231122T192907, start_date=20231122T210545, end_date=20231122T210545
[2023-11-22T21:05:45.597+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 51 for task execute_sql_script (column "tous âges" does not exist
LINE 54: INSERT INTO Age VALUES (0, "tous âges");
                                    ^
; 219)
[2023-11-22T21:05:45.614+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-22T21:05:45.634+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
